app/models/form526_submission.rb

```diff
@@ -93,12 +93,7 @@ def submit_with_birls_id_that_hasnt_been_tried_yet!(
   )
     untried_birls_id = birls_ids_that_havent_been_tried_yet.first
```

Comment:
No there can be any number of birls ids. How it works  (to the best of my knowledge, I did not write this...) is like a queue. Trying each one until successful or exhausted. 

IE if birls_ids starts as [100, 200, 300], it will queue a job with the first, to try, and the remaining, set as the ones that have not been tried yet. 

IE first run is `id: 100, untried: [200, 300]`

if it fails with 100, then itself queues `id: 200, untried: [300]` and so forth.

Reply:
I gottcha, ty for explaining